<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsNewHttpConnectionBase</class>
 <widget class="QDialog" name="QgsNewHttpConnectionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New Connection</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="mGroupBox">
     <property name="title">
      <string>Connection Details</string>
     </property>
     <layout class="QGridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="mWfsOptionsGroupBox">
        <property name="title">
         <string>WFS Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout1">
         <item row="0" column="0">
          <widget class="QLabel" name="lblVersion">
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="mWfsVersionDetectButton">
           <property name="text">
            <string>Detect</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QCheckBox" name="cbxWfsIgnoreAxisOrientation">
           <property name="text">
            <string>Ignore axis orientation (WFS 1.1/WFS 2.0)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblMaxNumFeatures">
           <property name="text">
            <string>Max. number of features</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cmbVersion"/>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="txtPageSize">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a number to limit the maximum number of features retrieved in a single GetFeature request when paging is enabled. If let to empty, server default will apply.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QCheckBox" name="cbxWfsInvertAxisOrientation">
           <property name="text">
            <string>Invert axis orientation</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblPageSize">
           <property name="text">
            <string>Page size</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblFeaturePaging">
           <property name="text">
            <string>Feature paging</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QComboBox" name="cmbFeaturePaging"/>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="txtMaxNumFeatures">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a number to limit the maximum number of features retrieved per feature request. If let to empty, no limit is set.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QCheckBox" name="cbxWfsUseGml2EncodingForTransactions">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This might be necessary on some &lt;span style=&quot; font-weight:600;&quot;&gt;broken&lt;/span&gt; ESRI map servers when using WFS-T 1.1.0.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Use GML2 encoding for transactions</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblVersion_2">
           <property name="text">
            <string>Preferred HTTP method</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="mComboHttpMethod"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="mTestConnectionButton">
        <property name="text">
         <string>&amp;Test Connection</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="mWmsOptionsGroupBox">
        <property name="title">
         <string>WMS/WMTS Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="13" column="0" colspan="2">
          <widget class="QCheckBox" name="cbxSmoothPixmapTransform">
           <property name="text">
            <string>Smooth pixmap transform</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QCheckBox" name="cbxWmsIgnoreAxisOrientation">
           <property name="text">
            <string>Ignore axis orientation (WMS 1.3/WMTS)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbTilePixelRatio"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblFeatureCount">
           <property name="text">
            <string>Maximum number of GetFeatureInfo results</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QCheckBox" name="cbxWmsInvertAxisOrientation">
           <property name="text">
            <string>Invert axis orientation</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCheckBox" name="cbxIgnoreGetFeatureInfoURI">
           <property name="text">
            <string>Ignore GetFeatureInfo URI reported in capabilities</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblDpiMode">
           <property name="text">
            <string>WMS DPI-&amp;Mode</string>
           </property>
           <property name="buddy">
            <cstring>cmbDpiMode</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QCheckBox" name="cbxWmsIgnoreReportedLayerExtents">
           <property name="text">
            <string>Ignore reported layer extents</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cmbDpiMode"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblTilePixelRatio">
           <property name="text">
            <string>WMTS server-side tile pixel ratio</string>
           </property>
           <property name="buddy">
            <cstring>cmbTilePixelRatio</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="cbxIgnoreGetMapURI">
           <property name="text">
            <string>Ignore GetMap/GetTile/GetLegendGraphic URI reported in capabilities</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="sbFeatureCount">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify a default value for FEATURE_COUNT when a new layer is created from this connection. &lt;/p&gt;&lt;p&gt;FEATURE_COUNT defines the maximum number of results returned by a GetFeatureInfo request, if not specified the server default value (usually 1) will be used.&lt;/p&gt;&lt;p&gt;Set to 0 to use server default: no FEATURE_COUNT parameter will be added to the request.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="specialValueText">
            <string>server default</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QgsHttpHeaderWidget" name="mHttpHeaders" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabel1_2">
           <property name="text">
            <string>Name</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtName">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Name of the new connection</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabel1">
           <property name="text">
            <string>URL</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtUrl">
           <property name="toolTip">
            <string>HTTP address of the Web Map Server</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="mAuthGroupBox">
        <property name="title">
         <string>Authentication</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QgsAuthSettingsWidget" name="mAuthSettings" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsHttpHeaderWidget</class>
   <extends>QWidget</extends>
   <header>qgshttpheaderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtUrl</tabstop>
  <tabstop>cmbVersion</tabstop>
  <tabstop>mWfsVersionDetectButton</tabstop>
  <tabstop>mComboHttpMethod</tabstop>
  <tabstop>txtMaxNumFeatures</tabstop>
  <tabstop>cmbFeaturePaging</tabstop>
  <tabstop>txtPageSize</tabstop>
  <tabstop>cbxWfsIgnoreAxisOrientation</tabstop>
  <tabstop>cbxWfsInvertAxisOrientation</tabstop>
  <tabstop>cbxWfsUseGml2EncodingForTransactions</tabstop>
  <tabstop>cmbDpiMode</tabstop>
  <tabstop>cmbTilePixelRatio</tabstop>
  <tabstop>sbFeatureCount</tabstop>
  <tabstop>cbxIgnoreGetMapURI</tabstop>
  <tabstop>cbxIgnoreGetFeatureInfoURI</tabstop>
  <tabstop>cbxWmsIgnoreReportedLayerExtents</tabstop>
  <tabstop>cbxWmsIgnoreAxisOrientation</tabstop>
  <tabstop>cbxWmsInvertAxisOrientation</tabstop>
  <tabstop>cbxSmoothPixmapTransform</tabstop>
  <tabstop>mTestConnectionButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsNewHttpConnectionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>813</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsNewHttpConnectionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>813</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
